src/interface/file.rs

```diff
@@ -268,18 +268,18 @@ impl EmulatedFileMap {
         // set the file equal to where were mapping the count and the actual map
         let _newsize = f.set_len((COUNTMAPSIZE + mapsize) as u64).unwrap();
 
-        let map : Vec::<u8>;
-        let countmap : Vec::<u8>;
+        let map : Vec<u8>;
+        let countmap : Vec<u8>;
 
         // here were going to map the first 8 bytes of the file as the "count" (amount of bytes written), and then map another 1MB for logging
         unsafe {
-            let map_addr = mmap(0 as *mut c_void, MAP_1MB, PROT_READ | PROT_WRITE, MAP_SHARED, f.as_raw_fd() as i32, 0 as i64);
+            let map_addr = mmap(0 as *mut c_void, MAP_1MB, PROT_READ | PROT_WRITE, MAP_SHARED, f.as_raw_fd() as i32, 0i64);
```

Comment:
Let's defer until after the [clippy issue](https://github.com/Lind-Project/lind_project/issues/373) is discussed.